os/board/rtl8721csm/src/component/common/api/wifi/wifi_conf.c

```diff
@@ -502,7 +502,7 @@ void restore_wifi_info_to_flash(void)
 		}
 
 		memcpy(wifi_data_to_flash.psk_essid, psk_essid[index], sizeof(wifi_data_to_flash.psk_essid));
-		if (strlen((char const*)psk_passphrase64) == 64) {
+		if (strlen((char const*)psk_passphrase64) == IW_WPA2_PASSPHRASE_MAX_SIZE) {

```

Comment:
I think you need to add `#include ieee80211.h` for `IW_WPA2_PASSPHRASE_MAX_SIZE`.